src/js/common/components/RequiredLoginView.jsx

```diff
@@ -129,7 +129,7 @@ class RequiredLoginView extends React.Component {
           view = <SystemDownView messageLine1="Sorry, our system is temporarily down while we fix a few things. Please try again later."/>;
         } else if (this.state.profileStatus === 'NOT_FOUND') {
           // If va_profile is "not found", we cannot find you in our system and we will show a, we can't find you message.
-          view = <SystemDownView messageLine1="We couldn't find your records with that information. Please call support at 1-855-574-7286."/>;
+          view = <SystemDownView messageLine1="We couldn't find your records with that information. Please call us for support." messageLine2="The vets.gov Help Desk 855-574-7286 is open Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET)"/>;
```

Comment:
Small tweaks:
"We couldn't find your records with that information. Please call the Vets.gov Help Desk at 1-855-574-7286. We're open Monday‒Friday, 8:00 a.m.‒8:00 p.m. (ET)."